### PR TITLE
Support adl pipeline

### PR DESCRIPTION
**Pull request by SparkSnail**

### CODE REVIEW

**Comment by SparkSnail** [comment]
@hao-howard-zhang @pw2393 I've add the pipeline for ADL, please help to review this pr, thanks.

File: test/config/examples/cifar10-pytorch-adl.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+searchSpacePath: ./cifar10_adl_search_space.json
+
+tuner:
+  builtinTunerName: Random
+assessor:
+  builtinAssessorName: Medianstop
+  classArgs:
+    optimize_mode: maximize
+trial:
+  codeDir: ../../../examples/trials/cifar10_pytorch
```
Comment (hao-howard-zhang) [inline test/config/examples/cifar10-pytorch-adl.yml:15]: Can we just set it as `/examples/trials/cifar10_pytorch`?

Comment (SparkSnail) [reply test/config/examples/cifar10-pytorch-adl.yml]: updated.

Comment (hao-howard-zhang) [reply test/config/examples/cifar10-pytorch-adl.yml]: Thanks. The rest LGTM.

**Review by liuzhe-lz — APPROVED** [review]

**Review by J-shang — APPROVED** [review]

**Pull request merged**